source/CCustomOpcodeSystem.cpp

```diff
@@ -602,7 +590,7 @@ namespace CLEO
 					{
 						if (CLEO_GetOperandType(thread) == DT_END) goto _ReadFormattedString_ArgMissing;
 
-						const char* str = ReadStringParam(thread, bufa, sizeof(bufa));
+						const char* str = OPCODE_READ_PARAM_STRING_BUFF(bufa, sizeof(bufa));

```

Comment:
These macros should be only used inside opcode handlers! Calling this function now may throw error message box with information about some invalid opcode and exit the function with OR_INTERUPT. Which if for sure not something that method returning bool or called from export should do.